index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <h2>John Omachi</h2>
+      <img src="./images/hamburger.png" alt="" />
+    </nav>
```

Comment:
- [ ] `required` I suggest you add the nav in a `header` 
- [ ] `optional` please dont the `alt` empty because its not good practice

Reply:
Nav has been added to a Header tag Successfully.
All empty strings in alt attribute of the image tag has been corrected

Reply:
[@mwanawabangona](https://github.com/mwanawabangona)